solutions/add_numbers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+"""
+A module for adding two numbers.
```

Comment:
Is this the only line to explain this module?

Reply:
Yes this is the only line.